### PR TITLE
update aa_recoder documentation

### PR DESCRIPTION
**Pull request by lnk1196**
- Changes name from SR4_class_recoder.py to aa_recoder.py
- Updates documentation for aa_recoder.py
- Updates nav_order for all utilities to maintain alphabetical order
- Updated index with new name and description
- Added new references

### CODE REVIEW

File: pages/utilites/backup_restoration.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: default
 title: backup_restoration.py
 parent: Utilities
-nav_order: 3
+nav_order: 5
```
Comment (robert-ervin-jones) [inline pages/utilites/backup_restoration.md:5]: I think this should be 4

File: pages/utilites/bipartition_examiner.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 layout: default
 title: bipartition_examiner.py
 parent: Utilities
-nav_order: 4
+nav_order: 6
```
Comment (robert-ervin-jones) [inline pages/utilites/bipartition_examiner.md:5]: I think this should be 5

**Review by robert-ervin-jones — CHANGES_REQUESTED** [review]
I think the nav orders on two are off. I left comments on the ones I think are wrong. Otherwise looks good

File: pages/utilites/aa_recoder.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- `-i`, `--input <matrix.fas|nex|phy>` Path to input matrix
+- `-re`, `--recoding_strat <strategy>` Desired amino acid recoding strategy
+	- Options: `SR4`, `SR6`, `KGB6`, `D6`, `D9`, `D12`, `D15`, `D18`
+
+Optional arguments:
+- `--in_format <format>` Input file format if not `fasta`
+  - Options: `fasta`, `phylip` (names truncated at 10 characters), `phylip-relaxed` (names are not truncated), or `nexus`
+  - Default: `fasta`
+- `-o`, `--output <out_dir>` Path to user-defined output directory
+  - Default: `./SR4_class_recoder_out_<M.D.Y>`
```
Comment (atice) [inline pages/utilites/aa_recoder.md:25]: The default output directory name needs changed to reflect the script name change.

File: pages/utilites/index.md [hunk 4]
```diff
@@ -13,6 +13,8 @@ While the above processes will allow users to develop a phylogenomic dataset and
 
 [`aa_comp_calculator.py`](https://thebrownlab.github.io/phylofisher-pages/utilities/aa-comp-calculator): Calculates amino acid composition and uses euclidean distances to hierarchically cluster these data, in order to examine if amino acid composition may bias the groupings that were inferred in a phylogenomic tree. See Brown et al. 2018 for an example.
 
+[`aa_recoder.py`](https://thebrownlab.github.io/phylofisher-pages/utilities/aa_recoder): To minimize phylogenetic saturation this tool recodes input supermatrix into one of eight schemes based on amino acid classification found in Susko and Rogers 2007, Kosiol et al. 2004, Feuda et al. 2017, and Hernandez and Ryan 2021..
```
Comment (atice) [inline pages/utilites/index.md:16]: Add a comma after saturation.
Remove the extra period after 2021.

**Review by atice — CHANGES_REQUESTED** [review]
A few changes required.

File: pages/utilites/backup_restoration.md (hunk 1 above)
Comment (lnk1196) [reply pages/utilites/backup_restoration.md]: Fixed

**Review by robert-ervin-jones — APPROVED** [review]

**Pull request merged**